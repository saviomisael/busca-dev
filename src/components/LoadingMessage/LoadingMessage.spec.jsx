import { LoadingMessage } from '.';

describe('<LoadingMessage />', () => {
  it('should renders component', () => {
    const { container } = global.renderTheme(<LoadingMessage />);

    expect(container.firstChild).toMatchSnapshot();
  });
});
