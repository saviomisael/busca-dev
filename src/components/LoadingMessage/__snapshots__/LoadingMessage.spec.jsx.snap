// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<LoadingMessage /> should renders component 1`] = `
.c0 {
  min-height: 100vh;
  max-width: 100vw;
  -webkit-filter: blur(1rem);
  filter: blur(1rem);
  position: relative;
}

<div
  class="c0"
/>
`;
